Map/MapManager.gd

```diff
@@ -26,3 +29,9 @@ static func create_map(map_floors_width: Array[int]) -> MapBase: ## Generates an
 		_grid[index_height].append_array(_padding)
 	_map.rooms = _grid
 	return _map as MapBase
+	
+func _ready():
+	current_map = MapManager.create_map(map_width_array)
```

Comment:
"The function "create_map()" is a static function but was called from an instance. Instead, it should be directly called from the type: "Node2D.create_map()"."
Well I thought using `MapManager` would work but apparently godot would rather have `Node2D`

Reply:
I think Godot may be a bit confused with calling a static function on itself. Using Node2D throws this error:
![image](https://github.com/Saplings-Projects/1M_sub/assets/5923102/7336872b-79c9-4f1a-91e5-53bfa4749bcd)

Reply:
yeah it's strange, it might indeed be confused because we call a static function from the class itself, not sure how to solve this so it doesn't throw a warning...

Reply:
MapManager is autoloaded correct? So could we make create_map not static and then do MapManager.create_map?

Reply:
Ya, was there a reason we had this static? It does work fine without static.

Reply:
can you `MapManager.create_map` if it's not static ? if yes then let's make it not static, but that means changing function calls to `¢reate_map` in relevant places

Reply:
Yeah AutoLoad lets you use that syntax

Reply:
well perfect then, let's call it directly